tests/filefinder.cpp

```diff
@@ -6,33 +6,33 @@
 #include "main_data.h"
 
 namespace {
-/*
 	void CheckIsRPG2kProject() {
-		EASYRPG_SHARED_PTR<FileFinder::ProjectTree> const
-			tree = FileFinder::CreateProjectTree(".");
+		std::shared_ptr<FileFinder::DirectoryTree> const
+			tree = FileFinder::CreateDirectoryTree(".");
 		assert(FileFinder::IsRPG2kProject(*tree));
+		FileFinder::SetDirectoryTree(tree);
 	}
-*/
+
 	void CheckIsDirectory() {
 		assert(FileFinder::IsDirectory("."));
 	}
 
 	void CheckEnglishFilename() {
-		assert(!FileFinder::FindImage("Backdrop", "castle").empty());
```

Comment:
The old test was doing a "touch" command for a RTP castle filename written in Japanese, maybe TestGame-2000 should have some filename in Japanese or vice-versa (use castle in Japanese in this string), to make this function name still meaningful. Or just rename the function :stuck_out_tongue_winking_eye:.